atlasdb-workload-server/src/test/java/com/palantir/atlasdb/workload/store/AtlasDbTransactionStoreTest.java

```diff
@@ -162,6 +166,29 @@ public void abortedTransactionsReturnEmpty() {
                 .isEmpty();
     }
 
+    @Test
+    public void keyAlreadyExistExceptionResultsInMaybeWitnessedTransaction() {
+        WriteTransactionAction writeTransactionAction =
+                WriteTransactionAction.of(TABLE_1, WORKLOAD_CELL_ONE, VALUE_ONE);
+        TransactionManager onlyThrowsKeyAlreadyExistsExceptionManager = spy(manager);
+        doAnswer(answer -> {
```

Comment:
nit: `Answer` is a functional interface with a method that takes an `InvocationOnMock`, so this param should be named `invocation` or `invocationOnMock`